packages/action-password-update/src/components/PasswordUpdateDialog/PasswordInput.js

```diff
@@ -10,6 +10,7 @@ const PasswordInput = props => (
       value={props.value}
       onChange={e => props.onChange(e.target.value)}
       readOnly={props.readOnly === true}
+      disabled={props.readOnly === true}
```

Comment:
I think we shouldn't use the `readOnly` prop to set both `readOnly` and `disabled`, because `readOnly` and `disabled` is not the same thing.

Do you want the password inputs to be read only or disabled? (see this post for a detailled expanation: http://stackoverflow.com/questions/7730695/whats-the-difference-between-disabled-disabled-and-readonly-readonly-for-ht)